examples/basic/outputs.tf

```diff
@@ -28,3 +28,8 @@ output "table_ids" {
   value       = module.bigtable.table_ids
   description = "List of table being provisioned"
 }
+
+output "project_id" {
```

Comment:
nit: it would be better to grab project in test via `GetTFSetupStringOutput("project_id")` but might be out of scope for this PR

Reply:
Thanks @bharathkkb - Will include in https://github.com/GoogleCloudPlatform/terraform-google-bigtable/pull/14